### PR TITLE
fahim update readme file again

### PR DESCRIPTION
**Pull request by fahimahmadi1985**
Please add my changes

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # Recap on Git
 
-This Readme file suppose to helps students to know how to use git for their final projects..
\ No newline at end of file
+This Readme file suppose to helps students to know how to use git for their final projects..
+
+Some other details
```
Comment (fahimahmadi1985) [inline readme.md:5]: This line of code edited by ahmadi as well

**Review by fahimahmadi1985 — COMMENTED** [review]
Please change your code

**Comment by fahimahmadi1985** [comment]
sorry this shouldn't be changed please use this in main branch

**Comment by fahimahmadi** [comment]
but my changes is better. let project owner decide about it.

**Review by fahimahmadi — APPROVED** [review]

**Pull request merged**